python/lsst/sims/catUtils/baseCatalogModels/GalaxyModels.py

```diff
@@ -60,9 +60,9 @@ class GalaxyObj(BaseCatalogObj):
                ('magNormBulge', 'magnorm_bulge'),
                ('magNormDisk', 'magnorm_disk'),
                ('magNormAgn', 'magnorm_agn'),
-               ('sedFilenameBulge', 'sedname_bulge', unicode, 40),
-               ('sedFilenameDisk', 'sedname_disk', unicode, 40),
-               ('sedFilenameAgn', 'sedname_agn', unicode, 40),
+               ('sedFilenameBulge', 'sedname_bulge', str, 40),
```

Comment:
In python 3, `str` is unicode-like.  `from builtins import str` imports a python 2 class that mimics that behavior

So: yes.  This will work, and it is still unicode-like.